### ISSUE TITLE
Error with get_cmap using Scatter visualization function

### ISSUE THREAD

**Issue by lisasoubirou** [anyoptimization/pymoo#603]
Hi,
I was trying to plot some results with the Scatter visualization function, but I got the following error:
<img width="547" alt="error" src="https://github.com/anyoptimization/pymoo/assets/125965505/258c6b0b-3f7e-4e41-b059-dae8c775162d">
I found this "issue" Re-add matplotlib.cm.get_cmap from matplotlib: https://github.com/matplotlib/matplotlib/pull/28355
Replacing self.cmap = matplotlib.cm.get_cmap(cmap) by self.cmap = plt.get_cmap(cmap) in the Plot.__init__() function solved my problem. I just wanted to point out this issue with the latest version of matplotlib (3.9)

**Comment by tacaswell** [comment]
Please use `matplotlib.colormaps.get_cmap()` instead.

**Comment by blankjul** [comment]
Does the current main branch work for you? There I have already merged a PR (Fixed the Plot Issue #593) where it is the following:

```
      # the colormap or the color lists to use
      if isinstance(cmap, str):
          self.cmap = matplotlib.pyplot.get_cmap(cmap)
      else:
          self.cmap = cmap
```
